resources/virt.go

```diff
@@ -82,6 +99,14 @@ func (obj *VirtRes) Init() error {
 		}
 		libvirtInitialized = true
 	}
+	if u, err := url.Parse(obj.URI); err != nil {
+		log.Printf("%s[%s]: Parsing URI failed: %s", obj.Kind(), obj.GetName(), obj.URI)
```

Comment:
Interesting point. Is there any situation though where it would fail, but still be a valid virt connection string? If so, then I agree with you, if not we're generating a failure message which might be confusing.